src/openassetio-python/package/openassetio/test/manager/apiComplianceSuite.py

```diff
@@ -589,39 +589,23 @@ def test_when_relation_unknown_then_no_pages_returned(self):
         an_unknown_rel = TraitsData({"🐠🐟🐠🐟"})
```

Comment:
[consider][minor] (Can't put a comment any closer) Docstring for test still contains "[Paged]"

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/1138/commits/1c33ef9d29913e4477834139f1c8a7339f57634a